CoffeeChat/Onboarding/DemographicsViewController.swift

```diff
@@ -16,8 +16,8 @@ class DemographicsViewController: UIViewController {
     private var fieldsEntered: [Bool] = [false, false, false, false] // Keep track of selection status of each field.
     private var fieldValues: [String: String] = [:] // Keep track of selected values
     // TODO: Update with networking values from backend (use states for now)
-    private let hometownSearchFields = ["Alabama", "Alaska", "Arizona", "Arkansas", "California", "Colorado", "Connecticut", "Delaware", "Florida", "Georgia", "Hawaii", "Idaho", "Illinois", "Indiana", "International", "Iowa", "Kansas", "Kentucky", "Louisiana", "Maine", "Maryland", "Massachusetts", "Michigan", "Minnesota", "Mississippi", "Missouri", "Montana", "Nebraska", "Nevada", "New Hampshire", "New Jersey", "New Mexico", "New York", "North Carolina", "North Dakota", "Ohio", "Oklahoma", "Oregon", "Pennsylvania", "Rhode Island", "South Carolina", "South Dakota", "Tennessee", "Texas", "Utah", "Vermont", "Virginia", "Washington", "West Virginia", "Wisconsin", "Wyoming"]
-    private let pronounSearchFields = ["She/Her/Hers", "He/Him/His", "They/Them/Theirs"]
+    private let hometownSearchFields = Constants.Options.hometownSearchFields
+    private let pronounSearchFields = Constants.Options.pronounSearchFields
```

Comment:
You haven't saved too many characters with this variable name, but you have introduced a class constant. If you don't use this variable too often, I would just say `Constants.Options. ...` each time.